bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractContainerImpl.java

```diff
@@ -105,6 +109,17 @@ protected void initBaseModel() {
         if (request != null && resource.isResourceType("wcm/foundation/components/responsivegrid")) {
             resGrid = request.adaptTo(ResponsiveGrid.class);
         }
+
+        if (request != null) {
```

Comment:
This logic will execute for all containers, can we make it specific to form container. You can add one more layer of abstraction specific to form container

Reply:
Ok, it makes sense. This logic is specific to form container so will add it there only.